WooCommerce/WooCommerceTests/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewModelTests.swift

```diff
@@ -293,6 +336,32 @@ final class BulkUpdateViewModelTests: XCTestCase {
         // Then
         XCTAssertTrue(onCancelButtonTappedInvoked)
     }
+
+    func test_less_than_100_variations_shows_warning() throws {
```

Comment:
nit: I think the names of these two new tests don't reflect what's being tested. It should be something like `test_less_than_100_variations_does_not_show_warning` and `test_more_than_100_variations_shows_warning`, right?

Reply:
lol I swapped the names 🤦